### PR TITLE
Override jsdom XMLHttpRequest

### PR DESCRIPTION
**Pull request by thrashr888**
The XMLHttpRequest in jsdom does not support local paths as urls. We can swap for one that does.

@apparentlymart

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var url = require('url');
 var angularcontext = require('angularcontext');
 var ngoverrides = require('./ngoverrides.js');
 var escapeHtml = require('escape-html');
+var XMLHttpRequest = require("local-xmlhttprequest").XMLHttpRequest;
```
Comment (apparentlymart) [inline lib/main.js]: Is there a `package.json` change to go with this?

Comment (thrashr888) [inline lib/main.js]: Ah crap. Yeah, I missed that one. The tests failed, too. I'll fix em up.

**Comment by apparentlymart** [comment]
Seems plausible apart from the missing `package.json` entry...

**Comment by apparentlymart** [comment]
Once tests are passing, LGTM.

**Pull request merged**